commandapi-core/src/main/java/dev/jorel/commandapi/StringTooltip.java

```diff
@@ -28,8 +35,58 @@
 public class StringTooltip implements IStringTooltip {
 
 	private final String suggestion;
-	private final String tooltip;
-	
+	private final Message tooltip;
+
+
+
+	/**
+	 * Constructs a StringTooltip with a suggestion and a string tooltip
+	 *
+	 * @param suggestion the suggestion to provide to the user
+	 * @param tooltip    the string tooltip to show to the user when they hover over the
+	 *                   suggestion
+	 * @return a StringTooltip representing this suggestion and tooltip
+	 */
+	public static StringTooltip ofString(String suggestion, String tooltip) {
```

Comment:
I like the naming of these methods (e.g. `ofString`, `ofMessage` etc.), but we need to retain an `of(String, String)` method for backward compatibility. (I'm a big fan of `ofString()`, please keep this and implement a deprecated `of(String, String)` method as well)

Reply:
997bbd9